README.md

```diff
@@ -114,6 +114,45 @@ after 'deploy:starting', 'sidekiq_systemd:quiet'
 after 'deploy:updated', 'sidekiq_systemd:stop'
 ```
 
+
+### Verify Ruby Versions
+
+There are three tasks available to check and verify the ruby version(s) installed and being used on the remote server:
+
+#### versions
+
+```
+bundle exec cap env ruby:versions
+=> host-1.stanford.edu,3.1.1,2.7.1,2.7.2,3.0.0,3.0.3,3.1.0,3.1.1,3.1.2
+=> host-2.stanford.edu,3.1.1,2.7.1,2.7.2,3.0.0,3.0.3,3.1.0,3.1.1,3.1.2
+```
+
+This comman seperated list is hostname,default ruby,installed versions. This list can then be uesd in automated deployment tools to report across hosts where version mismatches may occur.
+
```

Comment:
small typo (`comman` --> `comma`)

Reply:
It wouldn't be my PR without a few typos. Thanks!